hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/TestFileUtil.java

```diff
@@ -1493,6 +1495,73 @@ public void testReadSymlinkWithAFileAsInput() throws IOException {
     file.delete();
   }
 
+  /**
+   * Test that bytes are written out correctly to the local file system.
+   */
+  @Test
+  public void testWriteBytes() throws IOException {
+    setupDirs();
+
+    URI uri = tmp.toURI();
+    Configuration conf = new Configuration();
+    FileSystem fs = FileSystem.newInstance(uri, conf);
```

Comment:
just use FileSystem.get()

Reply:
This was copy & paste from other tests in this same class.  I can look at that though.

Reply:
as well as performance issues, you will leak filesystem instances